frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -142,6 +146,10 @@ public BranchOrCommitCellRendererComponent(
       IBranchItem branchItem = graphItem.asBranchItem();
       IGitMacheteBranch branch = branchItem.getBranch();
 
+      if (branch.isNonRootBranch()) {
+        setToolTipText(getSyncToParentStatusBasedTooltip(branch.asNonRootBranch()));
```

Comment:
Nit: use `ToolTip` (uppercase `T`) consistently with the existing API

Reply:
Hmmm not sure... but doesn't the tooltip just show **once** for each branch (so only when the mouse is hovered over the given row for the very first time)? @mkondratek do you observe the same behavior/glitch?

Reply:
@micpiotrowski: bumping the comment about tooltips showing up just once-twice :/

Reply:
I observed that if tooltip has the same content then is shown only once but if we hover another branch with other tooltip content then it's shown again

Reply:
![branch_tooltip](https://user-images.githubusercontent.com/56402877/89769711-e7dfb400-dafd-11ea-922c-3e773dc19457.gif)

Reply:
Ass agreed via Slack, we leave it as is for now + issue #500 created

Reply:
confirming